### PR TITLE
Universal wrap function

### PR DESCRIPTION
**Pull request by ivan-BRHM**
#43 and #33
created a function to let any sprite wrap.
Added it in .update for; Asteroid, PlayerShot and Player.
Can be considered to be added to UFOShot.

### CODE REVIEW

File: my_game.py [hunk 1]
```diff
@@ -46,6 +46,25 @@
...
 
+
+def wrap(sprite: arcade.Sprite):
+    """
+    if sprite is off-screen move it to the other side of the screen
+    """
+
+    if sprite.right < 0:
+        sprite.center_x += SCREEN_WIDTH
+
```
Comment (tobiasbp) [inline my_game.py:57]: Please make two else if blocks for x and y respectively.

**Review by tobiasbp — CHANGES_REQUESTED** [review]
Cool stuff. Minor change needed.

File: my_game.py (hunk 1 above)
Comment (ivan-BRHM) [reply my_game.py:57]: You mean elif?

Comment (tobiasbp) [reply my_game.py:57]: Yes!

**Review by tobiasbp — CHANGES_REQUESTED** [review]
Nice work. Merge it in.

**Comment by ivan-BRHM** [comment]
can't merge... Are you sure its approved?

**Review by tobiasbp — APPROVED** [review]

**Pull request merged**